wp-content/mu-plugins/pantheon/pantheon-page-cache.php

```diff
@@ -246,7 +313,6 @@ public function view_settings_page() {
 			}
 			</style>
 
-			<h3><?php _e( 'Default Time to Live (TTL)', 'pantheon-cache' ); ?></h3>
 			<form action="options.php" method="POST" class="ttl-form">
 				<?php settings_fields( self::SLUG ); ?>
 				<?php do_settings_sections( self::SLUG ); ?>
```

Comment:
@danielbachhuber since this page is doing more than the Default Time to Live setting, can the submit button be changed to "Save Changes"?

Reply:
@CdrMarks Yep, good catch. Updated with 811f1ec3